bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/package.scala

```diff
@@ -0,0 +1,8 @@
+package edu.gemini.spModel
+
+import scalaz._
+import Scalaz._
+
+package object target {
+  implicit val EqualSPTarget: Equal[SPTarget] = Equal.equalRef
+}
```

Comment:
Yeah, everything in the OT and program model was written assuming that targets use reference equality.  It makes sense in so much as having a mutable target in the first place makes sense.